Pipfile

```diff
@@ -41,6 +41,7 @@ sqlalchemy = "*"
 python-dotenv = "*"
 wget = "*"
 pillow = "*"
+pipfile = "*"
 
```

Comment:
Not sure what 'pipfile' is here or if we need it. It looks like a module called 'pipfile' was trying to be installed.